optional/vtd/CMakeLists.txt

```diff
@@ -22,13 +23,17 @@ add_library(vtd-object-lib STATIC
     src/scp_messages.cpp
     src/scp_transceiver.cpp
 )
+target_include_directories(vtd-object-lib
+  PRIVATE
+    protobuf::libprotobuf
+)
 target_link_libraries(vtd-object-lib
   PUBLIC
     vtd::api
     cloe::runtime
     cloe::models
     Boost::boost
-    open-simulation-interface::open-simulation-interface
+    open_simulation_interface::open_simulation_interface
```

Comment:
is OSI required at runtime? can this be a private build dependency?

Reply:
I'm not sure. I'd prefer to leave as is for now.